docs/docs/glamor.md

```diff
@@ -2,4 +2,4 @@
 title: Glamor
 ---
 
-CSS-in-JS library [Glamor](https://github.com/threepointone/glamor) is not actively maintained. the maintainer recommends using [Emotion](/docs/emotion/).
+CSS-in-JS のライブラリーである [Glamor](https://github.com/threepointone/glamor) は現在メンテナンスされておりません。 そのメンテナーは [Emotion](/docs/emotion/) の使用を勧めています。
```

Comment:
```suggestion
CSS-in-JS ライブラリ [Glamor](https://github.com/threepointone/glamor) は現在は積極的にメンテナンスされていません。メンテナーは [Emotion](/docs/emotion/) の使用を勧めています。
```

Reply:
It's not a big problem, but it's always good to have a uniform style of translation.
https://github.com/gatsbyjs/gatsby-ja/blob/master/docs/docs/creating-global-styles.md

Reply:
Of course. I'll be careful!